### PR TITLE
PR: add intro to NetworkPolicy

### PR DESCRIPTION
**Pull request by muse-sisay**
I was having a hard time grasping Network Policies. Empty selectors and null rules made it worse. So here are some gotcah's and rules I wish I knew before I delved into this repository. I hope this helps someone who is in a position I was .

### CODE REVIEW

**Review by boredabdel — COMMENTED** [review]
Added few comments. Mostly grammatical.

File: README.md [hunk 1]
```diff
@@ -17,6 +17,27 @@ If you are not familiar with Network Policies at all, I recommend reading my
...
+## NetworkPolicy Crash Course 
+NetworkPolicy works at layer 3 or 4 of OSI model and control what goes to and from a pod. 
+
+Here are some NetworkPolicy gotcha's  
+- An empty selector will match everything. For example `spec.podSelector: {}` will apply the policy to all pods in the current namespace.
+
+- Selectors can only select Pods that are in the same namespace as the NetworkPolicy. Eg. `spec.podSelector` of an ingress rule can only select pods in the same namespace the NetworkPolicy is deployed in. 
+
+Rules 
+- **RULE 1:** If no NetworkPolicy targets a pod, all traffic to and from all pods in all namespace is allowed. In other words all traffic is allowed unless there is one restricting it. Remember there is no deny rule in NetworkPolicy. You deny  a traffic by not mentioning it. "If you're not on the list you can't get in".
```
[the remark this answers is not in the document]
Comment (muse-sisay) [reply README.md:29]: Correct, that's what I meant to say.

**Comment by muse-sisay** [comment]
@boredabdel Thank you very much for taking the time to leave a review.

**Review by boredabdel — APPROVED** [review]
LGTM

**Pull request merged**